packages/react-composites/src/composites/CallWithChatComposite/CallWithChatComposite.tsx

```diff
@@ -99,6 +108,8 @@ type CallWithChatScreenProps = {
   callControls?: boolean | CallWithChatControlOptions;
   onFetchAvatarPersonaData?: AvatarPersonaDataCallback;
   onFetchParticipantMenuItems?: ParticipantMenuItemsCallback;
+  /* @conditional-compile-remove(file-sharing) */
+  fileSharing?: FileSharingOptions;
```

Comment:
We would still need conditional compilation because `FileSharingOptions` is being conditionally exported from ChatComposites :/
Any place that references it needs to be conditionally removed.